CONTRIBUTING.md

```diff
@@ -50,16 +50,17 @@ system already, but many of them are preinstalled on a Pivotal dev box.
 
 (Run all of these commands from the project root)
 
-ruby 2.1.2 
+ruby 2.1.2
 
-bundle
+bundler
 
-  $ gem install bundle
-    
-node and npm 
+  $ gem install bundler
+
+node and npm (at this time `nodejs` version `0.10` is required. Install
+[nvm](https://github.com/creationix/nvm) then use it to install node and npm.)
+
+  $ nvm install 0.10
```

Comment:
Gotcha. That makes sense. We should push contributors to use a more recent version of node, but we can do that down the line after we've vetted a newer version. I'll accept these doc changes as is.